README.md

```diff
@@ -4,9 +4,9 @@ OpenShift Application Platform
 [![GoDoc](https://godoc.org/github.com/openshift/origin?status.png)](https://godoc.org/github.com/openshift/origin)
 [![Travis](https://travis-ci.org/openshift/origin.svg?branch=master)](https://travis-ci.org/openshift/origin)
 
-This is the source repository for [OpenShift 3](https://openshift.github.io), based on top of [Docker](https://www.docker.io) containers and the
+This is the source repository for [OpenShift Origin](https://openshift.github.io), based on top of [Docker](https://www.docker.io) containers and the
```

Comment:
Updated, along with the other references to OpenShift that I was hesitant to change.